adapters/jwplayer/utils.go

```diff
@@ -0,0 +1,271 @@
+package jwplayer
+
+import (
+	"encoding/json"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/url"
+	"strconv"
+	"strings"
+	"text/template"
+)
+
+func ParseExtraInfo(v string) ExtraInfo {
+	var extraInfo ExtraInfo
+	if err := json.Unmarshal([]byte(v), &extraInfo); err != nil {
+		extraInfo = ExtraInfo{}
+	}
+
+	if extraInfo.TargetingEndpoint == "" {
+		extraInfo.TargetingEndpoint = "https://content-targeting-api.longtailvideo.com/property/{{.SiteId}}/content_segments?content_url=%{{.MediaUrl}}&title={{.Title}}&description={{.Description}}"
+	}
+
+	return extraInfo
+}
+
+func ParseBidderParams(imp openrtb2.Imp) (*openrtb_ext.ImpExtJWPlayer, error) {
```

Comment:
same comment as before, i think of utils as generic helpers that do not contain business logic, should these functions be moved to related modules?